src/lib/web3-helpers.ts

```diff
@@ -0,0 +1,33 @@
+export const loginSupportedChainIds: { [key: number]: string } = {
+  1: 'Ethereum Mainnet',
+  10: 'Optimism',
+  100: 'Gnosis',
+  137: 'Polygon',
+  250: 'Fantom Opera',
+  42220: 'Celo Mainnet',
+  42161: 'Arbitrum One',
+  43114: 'Avalanche C-Chain',
+};
+
+export async function switchNetwork(
```

Comment:
i suggest moving this to somewhere in src/utils (maybe src/utils/provider?) because it's very frontend/hook-specific

Reply:
good idea. done.